tests/data_out/write_vtu_with_pvtu_record_no_folder_01.mpirun=1.output

```diff
@@ -0,0 +1,31 @@
+
+DEAL:0::
+--------------------------------------------------------
+An error occurred in file <data_out_base.cc> in function
+    std::string dealii::DataOutInterface<dim, spacedim>::write_vtu_with_pvtu_record(const string&, const string&, unsigned int, ompi_communicator_t* const&, unsigned int, unsigned int) const [with int dim = 2; int spacedim = 2; std::string = std::__cxx11::basic_string<char>; MPI_Comm = ompi_communicator_t*]
```

Comment:
> std::string = std::__cxx11::basic_string<char>; MPI_Comm = ompi_communicator_t*

This is problematic as compiler and MPI details are part of the error message, so this will be fragile. If you remove the output, the tests becomes somewhat useless. :man_shrugging:

Reply:
You are right. What about this approach?